apps/site/src/lib/components/ValidatingForm/ValidatingForm.module.scss

```diff
@@ -7,3 +7,10 @@
 		display: block;
 	}
 }
+
+.unclickable {
+	button,
+	input {
+		pointer-events: none;
+	}
+}
```

Comment:
Issue: disabling pointer events does not necessarily disable interactions with the element (e.g. when focusing on the submit button and pressing enter).

Reply:
Thanks for pointing this out. I think maybe a better approach would be setting the `disabled` attribute of the button to false? We have access to the form element so we can access the button via the form's children attribute.

Reply:
how would we make sure that all the buttons in children are disabled though? Should we pass it as a prop and attach it to all the buttons?

Reply:
The only button we really need to disable (at least for the login form) is the submit one. We can access the button via [`getElementsByTagName`](https://developer.mozilla.org/en-US/docs/Web/API/Element/getElementsByTagName).

Reply:
As I mentioned in #170, we can move the submit button element to inside `ValidatingForm`

Reply:
but the form can also be submitted for example via pressing enter on the input field, would we need to have a fix for that as well?

Reply:
Hmm, we'll need to investigate more, some browsers will disable the form when it contains a button that is disabled, some browsers won't.

Reply:
there is also technically an "inert" attribute we can put that makes browsers ignore user input, but I don't know if that's appropriate here
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/inert

Reply:
We could also surround all of the form elements with a `fieldset` and then set the `disabled` attribute to `"disabled"`.

Reply:
I can't find it in the [Safari 16 Release Notes](https://developer.apple.com/documentation/safari-release-notes/safari-16-release-notes), but the issue appears to be fixed through WebKit/WebKit#3093.

Since most modern browsers will disable the form submission, I think we can proceed with the disabled button approach.

Reply:
slight issue: I'm not quite certain how to integrate button into ValidatingForm as the form's children are already passed in props and I can't edit the props or pass any more children into the form

Reply:
nevermind I think I figured it out